ql/src/java/org/apache/hadoop/hive/ql/ddl/view/materialized/alter/rebuild/AlterMaterializedViewRebuildAnalyzer.java

```diff
@@ -480,8 +515,6 @@ protected ASTNode fixUpAfterCbo(ASTNode originalAst, ASTNode newAst, CalcitePlan
         fixUpASTAggregateInsertDeleteIncrementalRebuild(fixedAST, getMaterializedViewASTBuilder());
         return fixedAST;
       case JOIN_INSERT_DELETE_REBUILD:
```

Comment:
why do we need an empty `case` for JOIN_INSERT_DELETE_REBUILD?

Reply:
removed by #5053